exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -0,0 +1,36 @@
+# Instructions
+
+In this exercise you will be building error handling for a simple integer calculator. To make matters simple, methods for calculating addition, multiplication and division are provided.
```

Comment:
Style-wise: can we put all new sentences on their own line so that diffs are cleaner if we need to update them?

For example:

```
In this exercise you will be building error handling for a simple integer calculator.
To make matters simple, methods for calculating addition, multiplication and division are provided.
```

Reply:
Ahh! Okay. I kind of just copied the readme from csharp one and made changes here and there. I will make the changes asked in the following comments. Also for intoduction.md is there a reference to writing it? And the configlet errors are ONLY due to missing files right?

Reply:
> Also for intoduction.md is there a reference to writing it?

Nothing more formal than what is written in https://exercism.org/docs/building/tracks/concept-exercises. There is a dedicated section for ".docs/introduction.md" that details it.

> And the configlet errors are ONLY due to missing files right?

I mean, it will tell you whatever errors it is finding. Right now the errors are:

```
Missing file:
/home/runner/work/java/java/exercises/concept/calculator-conundrum/.docs/hints.md

Missing file:
/home/runner/work/java/java/exercises/concept/calculator-conundrum/.docs/introduction.md

The `files` array contains value `src/test/java/CalculatorConundrumTest.java` but `/home/runner/work/java/java/exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java` could not be found:
/home/runner/work/java/java/exercises/concept/calculator-conundrum/.meta/config.json
```

So the third one there is complaining about the contents of `.meta/config.json` which is because you put it under src/test rather than src/test/java.